src/users/views.py

```diff
@@ -158,6 +160,50 @@ def post(self, request):
         profile.save()
         return Response({'profile_id': profile_id})
 
+class SendPasswordReset(APIView):
+    permission_classes = tuple()
+    def post (self, request):
+        email = request.data['username']
+        user = User.objects.get(username=email)
+
+        profile_id = user.profile.id
+        profile = Profile.objects.get(id=profile_id)
+
+        profile.password_reset_code = Profile.generate_password_reset_code()
+        profile.save()
+        url = 'https://bquest.ucladevx.com/password?code='
+        if settings.DEBUG:
+            url = 'http://localhost:8000/password?code='
+        
+        from_email =  Email('noreply@bquest.ucladevx.com')
+        to_email = Email(email)
+        subject = 'BQuest User Password Reset'
+        reset_link = url + profile.password_reset_code
+        content = Content('text/html', 'N/A')
+        mail = Mail(from_email, subject, to_email, content)
+        mail.personalizations[0].add_substitution(Substitution('password_reset_link', reset_link))
+        mail.template_id = PASSWORD_RESET_TEMPLATE
+
+        sg = sendgrid.SendGridAPIClient(apikey=settings.SENDGRID_API_KEY)
+        response = sg.client.mail.send.post(request_body=mail.get())
+        if not (200 <= response.status_code < 300):
+            raise ValidationError({'status_code': response.status_code})
```

Comment:
can you say `sendgrid_status_code` instead of `status_code`?  would make it easier to debug in the future